ga4gh/datamodel/genotype_phenotype.py

```diff
@@ -0,0 +1,467 @@
+"""
+Module responsible for translating g2p data into GA4GH native
+objects.
+"""
+
+import ga4gh.protocol as protocol
+import ga4gh.exceptions as exceptions
+
+import rdflib
+from rdflib import URIRef
+from rdflib import Graph
+
+
+class G2PDataset:
+    """
+    an rdf object store
+    """
+
+    # keep the graph as a static variable
+    g = None
```

Comment:
Single letter variables are generally frowned upon around here. They're only really considered OK when used in things like list comprehensions or lambda expressions. Definitely not good as a static class variable though. Something like 
```python
rdfGraph = rdflib.Graph()
```
is much easier to folllow.